flight/PiOS/Common/pios_mpu.c

```diff
@@ -0,0 +1,1090 @@
+/**
+ ******************************************************************************
+ * @file       pios_mpu.c
```

Comment:
Or maybe pios_invensense or pios_invensensemotion, as their new lineup has dropped the mpu prefix but is largely compatible.